boards/xtensa/lilygo_t3/doc/index.rst

```diff
@@ -0,0 +1,216 @@
+.. _liligo_t3:
```

Comment:
```suggestion
.. _lilygo_t3:
```

Reply:
changed while updating the image, thanks!